setup/azurecmdfiles.wxi

```diff
@@ -665,6 +665,9 @@
                         <Component Id="cmp0BDDC10BCAFF9EC25710B20367203609" Guid="*">
                             <File Id="fil1B9798D9FA9E8F9FC7A4235BCFAB93E4" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.Compute\AzureRM.Tags.psd1" />
                         </Component>
+                        <Component Id="cmpA3F52D8426DFD1BC61236CCBAAB1BFD1" Guid="*">
```

Comment:
These items kept showing up in this file even if I started from a clean repo. Are they missed from previous commits/PRs or is there another way to clean them up (besides manually)?